sessionctx/variable/sysvar.go

```diff
@@ -1861,6 +1861,18 @@ var defaultSysVars = []*SysVar{
 	{Scope: ScopeNone, Name: "version_compile_os", Value: runtime.GOOS},
 	{Scope: ScopeNone, Name: "version_compile_machine", Value: runtime.GOARCH},
 	{Scope: ScopeNone, Name: TiDBAllowFunctionForExpressionIndex, ReadOnly: true, Value: collectAllowFuncName4ExpressionIndex()},
+	{Scope: ScopeSession, Name: "rand_seed1", Type: TypeInt, Value: "0", skipInit: true, MaxValue: math.MaxInt32, SetSession: func(s *SessionVars, val string) error {
```

Comment:
Our convention for supported sysvars is to use a const for the name. It looks like it is broken above for version_compile_os, but it is mostly consistent in sysvar.go otherwise.

Reply:
Done, PTAL @morgo